src/chunk.c

```diff
@@ -1013,6 +1006,18 @@ chunk_create_table_after_lock(Chunk *chunk, Hypertable *ht)
 								  chunk->fd.id,
 								  chunk->table_id);
 	}
+}
+
+static Oid
+chunk_create_table(Chunk *chunk, Hypertable *ht)
+{
+	/* Create the actual table relation for the chunk */
+	const char *tablespace = ts_hypertable_select_tablespace_name(ht, chunk);
+
+	chunk->table_id = ts_chunk_create_table(chunk, ht, tablespace);
+
+	if (!OidIsValid(chunk->table_id))
```

Comment:
If this failure case is even possible, it might make sense to move this into `ts_chunk_create_table` immediately after`DefineRelation`. It isn't clear that that function can even return `InvalidOid`, so we might be able to get rid of this check altogether.

Reply:
Correct, it should not be possible. Changed to assertion for now.